src/ImageSharp/Formats/AnimationUtilities.cs

```diff
@@ -0,0 +1,190 @@
+                        int end = i + (2 - (BitOperations.LeadingZeroCount((uint)m) / size));
+                        left = Math.Min(left, start);
+                        right = Math.Max(right, end);
+                        hasRowDiff = true;
+                        hasDiff = true;
+                    }
+
+                    // Replace the pixel value with the replacement if the full pixel is matched.
+                    Vector256<float> r = Avx.BlendVariable(c, replacement256, mask.AsSingle());
+                    Unsafe.Add(ref resultBase, x) = r;
+
+                    x++;
+                    i += 2;
+                    remaining -= 2;
+                }
+            }
+
+            for (i = remaining; i > 0; i--)
+            {
+                x = (uint)(length - i);
+
+                Vector4 p = Unsafe.Add(ref Unsafe.As<Vector256<float>, Vector4>(ref previousBase), x);
+                Vector4 c = Unsafe.Add(ref Unsafe.As<Vector256<float>, Vector4>(ref currentBase), x);
+                ref Vector4 r = ref Unsafe.Add(ref Unsafe.As<Vector256<float>, Vector4>(ref resultBase), x);
+
+                if (p != c)
+                {
+                    r = c;
+
+                    // If is diff is found, the left side is marked by the min of previously found left side and the diff position.
+                    // The right is the max of the previously found right side and the diff position + 1.
+                    left = Math.Min(left, (int)x);
+                    right = Math.Max(right, (int)x + 1);
+                    hasRowDiff = true;
+                    hasDiff = true;
+                }
+                else
+                {
+                    r = replacement;
+                }
+            }
+
+            if (hasRowDiff)
+            {
+                if (top == int.MinValue)
+                {
+                    top = y;
+                }
+
+                bottom = y + 1;
+            }
+
+            PixelOperations<TPixel>.Instance.FromVector4Destructive(configuration, result, resultFrame.DangerousGetPixelRowMemory(y).Span, PixelConversionModifiers.Scale);
+        }
+
+        Rectangle bounds = Rectangle.FromLTRB(
+            left = Numerics.Clamp(left, 0, resultFrame.Width - 1),
+            top = Numerics.Clamp(top, 0, resultFrame.Height - 1),
+            Numerics.Clamp(right, left + 1, resultFrame.Width),
+            Numerics.Clamp(bottom, top + 1, resultFrame.Height));
+
+        // Webp requires even bounds
+        if (clampingMode == ClampingMode.Even)
+        {
+            bounds.Width = Math.Min(resultFrame.Width, bounds.Width + (bounds.X & 1));
+            bounds.Height = Math.Min(resultFrame.Height, bounds.Height + (bounds.Y & 1));
+            bounds.X = Math.Max(0, bounds.X - (bounds.X & 1));
+            bounds.Y = Math.Max(0, bounds.Y - (bounds.Y & 1));
+        }
+
+        return new(hasDiff, bounds);
```

Comment:
```suggestion
        return (hasDiff, bounds);
```
For the value tuples explicit `new` isn't needed.

(I rather find the `new` here confusing, because I'm think what type was it -- in contrast w/o the new it's like "ah, a value tuple")